projects/plugins/jetpack/extensions/blocks/donations/donations.php

```diff
@@ -188,6 +195,52 @@ function render_block( $attr, $content ) {
 	);
 }
 
+/**
+ * Get the default texts for the block.
+ *
+ * @return array
+ */
+function get_default_texts() {
+	return array(
+		'chooseAmountText' => __( 'Choose an amount', 'jetpack' ),
+		'customAmountText' => __( 'Or enter a custom amount', 'jetpack' ),
+		'extraText'        => __( 'Your contribution is appreciated.', 'jetpack' ),
+		'oneTimeDonation'  => array(
+			'heading'    => __( 'Make a one-time donation', 'jetpack' ),
+			'buttonText' => __( 'Donate', 'jetpack' ),
+		),
+		'monthlyDonation'  => array(
+			'heading'    => __( 'Make a monthly donation', 'jetpack' ),
+			'buttonText' => __( 'Donate monthly', 'jetpack' ),
+		),
+		'annualDonation'   => array(
+			'heading'    => __( 'Make a yearly donation', 'jetpack' ),
+			'buttonText' => __( 'Donate yearly', 'jetpack' ),
+		),
+	);
+}
+
+/**
+ * Make default texts available to the editor.
+ */
+function load_editor_scripts() {
+	// Only relevant to the editor right now.
+	if ( ! is_admin() ) {
+		return;
+	}
+
+	$data = array(
+		'defaultTexts' => get_default_texts(),
+	);
+
+	wp_add_inline_script(
```

Comment:
Make texts available to the editor.

Reply:
If you're going to do it this way rather than using `@wordpress/i18n`, why not use `wp_localize_script()` instead?